app/src/components/sections/ExpenseAccountView/ApprovedExpensesSection.vue

```diff
@@ -93,7 +87,7 @@ const {
 //#region Funtions
 const init = async () => {
```

Comment:
You can remove this function and call directly refetchExpenseAccountGetOwner
And you don't need to call in on mount. 
It's called by default.